.github/workflows/helm-test.yml

```diff
@@ -121,6 +124,47 @@ jobs:
       - name: List Services
         run: |
           minikube kubectl -- get services
+      - name: Service healthcheck (:30888/restconf/operations)
+        run: |
+          minikube_ip=$(minikube ip)
+          curl --user admin:admin -H "Content-Type: application/json" --insecure http://$minikube_ip:30888/restconf/operations
+      - name: Uninstall deployment
+        run: |
+          helm del ${{ github.event.inputs.app-name }} --purge
+          sleep 25 # Wait for helm to del deployment
```

Comment:
Try to implement check if uninstall already finished (for example get pods and check if RNC ones are still present or not)